independent-publisher-connectors/Google Chart QR Code/readme.md

```diff
@@ -0,0 +1,26 @@
+# Google Chart QR code API
+
+Google Charts has an API option to generate QR codes from data input.
+Based on parameters you can specify the maximum amount of data that's lost from the image to be able to scan it.
+Several sizes are also supported.
+
+# Publisher: Iona Varga
+[My Profile](https://www.linkedin.com/in/iona-v-641487191/)
+
+# Prerequisites
+This connector doesn't require authentication. It's made available for free by Google.
+
+# Operations
+This connector only has the Generate QR code action
+
+Generate QR Code: Create a QR code based around a few parameters and some data input.
```

Comment:
@Ionavoss All files should be free of grammatical and spelling errors. in the description you have "Create a QR code based around a few...", maybe you meant to say "Create a QR code based **on** a few..."? Can you please fix?